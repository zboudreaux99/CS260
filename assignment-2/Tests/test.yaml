test:
  timelimit: 10
  launch: ./tally
  params: '@T/albums.csv @T/requests.csv'
  stderr: false
dueDate: 12/06/2024
latePenalties: [10, 20, 100]
build:
  command: make
  weight: 1
  studentFiles:
    include: ["playlist.h", "playlist.cpp"]
  instructorFiles:
    exclude: ["playlist.h", "playlist.cpp"]
