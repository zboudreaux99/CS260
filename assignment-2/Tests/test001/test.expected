Read 83 songs.
Processed 2 playlist items.

The Great Gig In The Sky Pink Floyd          Dark Side Of The Moon    4 04:49
Lucy In the Sky With Dia Elton John          Captain Fantastic and th 11 06:17
Lucy In The Sky With Dia The Beatles         Sgt. Peppers Lonely Hear 3 03:26
Total: 14:32

